keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -15,20 +17,37 @@ def __init__(
         offset=None,
         norm_mean=[0.485, 0.456, 0.406],
         norm_std=[0.229, 0.224, 0.225],
+        bounding_box_format="yxyx",
         **kwargs,
     ):
-        super().__init__(**kwargs)
+        super().__init__(image_size=image_size, **kwargs)
+        self.resizing_bbox = keras.layers.Resizing(
+            height=image_size[0] if image_size else None,
+            width=image_size[1] if image_size else None,
+            bounding_box_format=bounding_box_format,
+            crop_to_aspect_ratio=self.crop_to_aspect_ratio,
+            pad_to_aspect_ratio=self.pad_to_aspect_ratio,
+            interpolation=self.interpolation,
+            data_format=self.data_format,
+            dtype=self.dtype_policy,
+            name="resizing_bbox",
+        )
+
         self.image_size = image_size
         self.scale = scale
         self.offset = offset
         self.norm_mean = norm_mean
         self.norm_std = norm_std
-        self.built = True
+        self.bounding_box_format = bounding_box_format
 
     @preprocessing_function
-    def call(self, inputs):
-        # TODO: https://github.com/keras-team/keras-hub/issues/1965
-        x = inputs
+    def call(self, x, y=None, sample_weight=None):
+        if y is not None:
```

Comment:
I don't think we need two resizing layers here (which is also just kind of confusing).

Wouldn't it work to make one `self.resizing` layer above, and then...

```python
if y is not None:
    inputs = self.resizing({"images": x, "bounding_boxes": y})
    x = inputs["images"]
    y = inputs["bounding_boxes"]
else:
    x = self.resizing(x) 
```